Library/SPIClient.m

```diff
@@ -1386,40 +1417,40 @@ - (void)onSpiConnectionStatusChanged:(SPIConnectionState)newConnectionState {
  */
 - (void)startPeriodicPing {
     __weak __typeof(&*self) weakSelf = self;
-
+    
     _pingTimer = [[SPIRepeatingTimer alloc] initWithQueue:"com.assemblypayments.ping"
                                                  interval:0
                                                     block:^{
```

Comment:
This all seems to just be whitespace change according to the github highlighting, so i'll not look too deeply